apps/bfd-model/bfd-model-dsl-codegen/bfd-model-dsl-codegen-plugin/src/test/resources/GenerateCsvWritersFromDslMojoIT/expected/DMEClaimCsvWriter.java

```diff
@@ -0,0 +1,34 @@
+package gov.cms.test;
+
+import java.lang.Object;
+import java.lang.String;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public final class DMEClaimCsvWriter {
```

Comment:
Docs missing here too; looked into this and checkstyle is missing this cause this java file is in test/resources. IMO source files shouldnt be in resources; if this is a utility class to create expected responses, it should still be under test/java, unless there is some compiler reason this is required to be here.

Reply:
Nothing missing here.  This is an expected output file for the test.